### DISCUSSION TITLE
Summary of similar projects

### DISCUSSION THREAD

**Discussion by downdawn** [fastapi-practices/fastapi_best_architecture#211] [General]
python

[https://github.com/vvandk/kinit](https://github.com/vvandk/kinit)
[https://github.com/mizhexiaoxiao/vue-fastapi-admin](https://github.com/mizhexiaoxiao/vue-fastapi-admin)

go

[https://github.com/go-admin-team/go-admin](https://github.com/go-admin-team/go-admin)
[https://github.com/flipped-aurora/gin-vue-admin](https://github.com/flipped-aurora/gin-vue-admin)

**Comment by wu-clan** [comment]
I will add to this later

**Comment by wu-clan** [comment]
please check: #214
